nautobot/utilities/tests/test_paginator.py

```diff
@@ -78,3 +79,28 @@ def test_enforce_max_page_size(self):
             self.assertHttpStatus(response, 200)
             self.assertEqual(response.context["paginator"].per_page, 10)
             self.assertEqual(len(response.context["table"].page), 10)
+        with self.subTest("query parameter with per_page exceeding max page, shows warning"):
```

Comment:
Instead of adding a new subtest, would it be appropriate to add warning message checks in any of the existing subtests above?